src/docs/admin/references/NodeSourceInfrastructureReference.adoc

```diff
@@ -357,8 +357,70 @@ now be added using the *Create Node Source* panel in the Resource Manager or the
     $ PROACTIVE_HOME/bin/proactive-client --createns vmware -infrastructure org.ow2.proactive.resourcemanager.nodesource.infrastructure.VmwareInfrastructure username password endpoint rmHostname connectorIaasURL image ram cores vmusername vmpassword numberOfInstances numberOfNodesPerInstance downloadCommand additionalProperties
 
 
-WARNING: When the Scheduler is stopped (without removing the created infrastructure), the VmWare instances will not be terminated. And when the Scheduler is restarted, the infrastrucutre will be re-configured as per previous settings. If the instances were delete at the VmWare server side, they will be re-created and re-configured.
+WARNING: When the ProActive server is stopped (without removing the created infrastructure), the VmWare instances will not be terminated. And when the ProActive server is restarted, the infrastrucutre will be re-configured as per previous settings. If the instances were delete at the VmWare server side, they will be re-created and re-configured.
 
+===== GCE Infrastructure
+
+Google Compute Engine, aka GCE, delivers virtual machines running on Google's infrastructure.
+To use GCE virtual machines as computing nodes, a specific GCE Infrastructure needs to be created using the Resource Manager.
+This section describes briefly how to make it.
+
+First, To use the GCE Infrastructure in the Resource Manager, proper Google Cloud credentials are needed for an authentification to the Google Cloud Platform.
+To obtain them, you can take the following steps:
+
+1. Go to the https://console.developers.google.com[Developer Console].
+2. Choose your project.
+3. Choose _APIs & Services_ > _Credentials_.
+4. Under the tab _Credentials_, Click _Create credentials_ > _Service account key_.
+5. Select the option _New service account_, then fill in your service account name, choose _Compute Engine_ > _Compute Admin_ as "role", choose _JSON_ as "Key type".
+6. Click the button _Create_, a JSON file for the created service account will be downloaded.
+
+For more information regarding Google Cloud service accounts see
+https://cloud.google.com/compute/docs/access/service-accounts[Google Cloud Service Accounts].
+
+
+Now, you are ready to create a new node source of the type _GCE Infrastructure_.
+The downloaded JSON file is used to fill in the properties *gceCredential* to perform the basic Google Cloud Platform authentification.
```

Comment:
which JSON file?

Reply:
Changed to "The downloaded JSON file **of Google Cloud credentials** is used to..."